### ISSUE TITLE
Silver Mechamaton

### ISSUE THREAD

**Issue by Cruuk** [Tslat/Advent-Of-Ascension#268]
**Describe/name the existing feature this suggestion relates to**
Mechamaton.

**Describe the change or feature you think would be good to see**
One day (less than an hour ago), I was looking at the Mechamaton page and I saw this.
![image](https://user-images.githubusercontent.com/38159897/51808495-68ef9700-2249-11e9-8007-ef906f6723df.png)
Suddenly, an idea came to my head. What if the Silvro hills biome had a silver variation of the Mechamaton?
Basically any Mechamatons that spawn in Silvro hills (when it's re-added) would have a silver color instead of the usual gold color. Not sure how difficult it would be coding-wise but I just thing it would be neat.

The reason I'm only suggesting this for the Mechamaton and not the other Iromine mobs is because it would give it more parallels to the Automaton, and I think that would be pretty neat...
